st2tests/st2tests/fixtures/packs_1/BUILD

```diff
@@ -0,0 +1,3 @@
+packs_glob(
+    name="all_packs",
+)
```

Comment:
Nothing depends on this (and the one in packs_invalid) directly right now. We will need both to include all the packs in the wheel.